app/src/main/java/ar/com/wolox/android/training/ui/login/LoginFragment.java

```diff
@@ -17,6 +26,9 @@ public static LoginFragment newInstance() {
 
     @Override
     public void init() {
+        termsConditions = getView().findViewById(R.id.vTermsConditions);
+        logInButton = getView().findViewById(R.id.vLogInButton);
+        signUpButton = getView().findViewById(R.id.vSignUpButton);
```

Comment:
This is is totally fine for now. But later we will see nicer ways to reference to view items. (@igsosa92 just in case you go through this and I'm not available)